macaroon/Classes/Views/Configuration/Layout/LayoutConvertible.swift

```diff
@@ -0,0 +1,7 @@
+// Copyright © 2019 hipolabs. All rights reserved.
+
+import Foundation
+
+public protocol LayoutConvertible {}
```

Comment:
Either make this one `LayoutGuideConvertible` or `associatedtype Layout` in `ViewComposable`. They should be named in the same context.